src/overlay/OverlayManager.h

```diff
@@ -90,6 +90,8 @@ class OverlayManager
         return recvFloodedMsgID(msg, peer, msgID);
     }
 
+    virtual void recvTransaction(Hash const& txHash, Peer::pointer peer) = 0;
```

Comment:
Please document this function

Reply:
Done (I also moved transaction processing to OverlayManager, as I don't think that functionality really belongs to Peer. At a high level, I think it makes more sense for Peer to just pass inbound messages to higher-level abstractions that will perform the business logic)